src/DemoUI/src/Views/resources/css/index.css

```diff
@@ -0,0 +1,1865 @@
+    color: #2095F2 !important;
+    border-right: 1px solid #dcdcdc !important;
+}
+
+.pl-table-header-wrapper th {
+    border-bottom: 1px solid #dcdcdc;
+}
+
+.pl-table-row-method-title h2 {
+    margin: 0 0 10px 0 !important;
+    padding: 0 !important;
+    color: #5d5d5d !important;
+    font-size: 14px;
+    font-weight: bold;
+}
+
+.pl-table-row-method-title p {
+    margin: 0 !important;
+    padding: 0 !important;
+    color: #5d5d5d !important;
+    font-size: 10px;
+}
+
+.pl-table-row-method-select, .pl-table-row-method-select *, .pl-table-row-method-logo,
+.pl-table-row-method-delivery-type, .pl-table-row-method-type,
+.pl-table-header-select, .pl-table-header-transit, .pl-table-header-carrier,
+.pl-table-header-type, .pl-table-header-origin, .pl-table-header-destination {
+    text-align: center !important;
+}
+
+.pl-tbody tr.pl-table-row-wrapper:last-child td {
+    border-bottom: 1px solid #dcdcdc !important;
+}
+
+.pl-tbody tr:first-child td {
+    border-top: none !important;
+}
+
+.pl-tbody tr,
+.pl-tbody td {
+    border-collapse: separate;
+}
+
+.pl-price-indicator {
+    display: none;
+}
+
+.pl-price-indicator.selected {
+    display: block;
+}
+
+.pl-method-logo {
+    max-height: 40px;
+    max-width: 100px;
+}
+
+.pl-method-pudo-icon-wrapper {
+    text-align: center !important;
+    color: #5d5d5d !important;
+    font-size: 10px !important;
+    max-width: 100px;
+    line-height: 15px;
+    margin: auto;
+}
+
+.pl-method-pudo-icon-wrapper .pl-pudo-pickup {
+    display: none;
+}
+
+.pl-method-pudo-icon-wrapper.pl-pickup .pl-pudo-dropoff {
+    display: none;
+}
+
+.pl-method-pudo-icon-wrapper.pl-pickup .pl-pudo-pickup {
+    display: block;
+}
+
+.pl-pudo-pickup svg {
+    display: block;
+    margin: auto;
+}
+
+.pl-pudo-dropoff svg {
+    display: block;
+    margin: auto;
+}
+
+.pl-table-row-arrow-wrapper {
+    width: 40px !important;
+}
+
+.pl-table-row-arrow svg {
+    fill: #2095F2 !important;
+    padding-left: 20px;
+}
+
+.pl-dashboard-modal-wrapper {
+    position: absolute;
+    top: -20px;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    padding: 20px;
+    background-color: rgba(0, 0, 0, 0.4);
+    z-index: 450;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.pl-dashboard-modal-wrapper.hidden {
+    display: none !important;
+}
+
+.pl-dashboard-modal {
+    background: white;
+    color: #d5d5d5;
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    padding: 20px 70px;
+    border-radius: 5px;
+    position: relative;
+    margin: 0 auto;
+    height: 100%;
+}
+
+.pl-increased-padding {
+    padding-top: 60px;
+}
+
+.pl-modal-button-wrapper {
+    padding: 0 70px 10px 70px;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    border-top: 1px solid;
+}
+
+.pl-modal-button-wrapper > button {
+    float: right;
+}
+
+.pl-button-separator {
+    margin-right: 10px !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)